src/core/public/chrome/constants.ts

```diff
@@ -31,3 +31,9 @@
 export const OPENSEARCH_DASHBOARDS_ASK_OPENSEARCH_LINK = 'https://forum.opensearch.org/';
 export const GITHUB_CREATE_ISSUE_LINK =
   'https://github.com/opensearch-project/OpenSearch-Dashboards/issues/new/choose';
+
+export enum RightNavigationOrder {
+  // order of dev tool should be after advance settings
+  Settings = 1,
+  DevTool = 2,
```

Comment:
nit: the value could be 10, 20 and leave some gaps between them so that we can add more between settings and devtool without change the value.

Reply:
That makes sense. Updated.